compiler/luci/service/src/CircleShapeInferenceRule.cpp

```diff
@@ -1691,7 +1691,51 @@ loco::NodeShape infer_bcq_gather(const luci::CircleBCQGather *node)
   return loco::NodeShape{output_shape};
 }
 
+loco::NodeShape infer_bidirectionalsequencelstm(const luci::CircleBidirectionalSequenceLSTM *node)
```

Comment:
please put this block after `infer_output`

Reply:
will address this, thanks.